Makefile.build

```diff
@@ -259,6 +259,7 @@ CAMLP5DEPS:=grammar/grammar.cma
 CAMLP5USE=pa_extend.cmo q_MLast.cmo pa_macro.cmo -D$(CAMLVERSION)
 
 PR_O := $(if $(READABLE_ML4),pr_o.cmo,pr_dump.cmo)
+# XXX unused but should be used for mlp files
```

Comment:
Can you remind me what mlp files are? Weren't they used just for camlp4 <-> camlp5 compatibility?

Reply:
camlp4 files that don't depend on VERNAC EXTEND &co (ie the ones used to define it)